Code/GraphMol/CIPLabeler/CIPLabeler.cpp

```diff
@@ -0,0 +1,193 @@
+//
+//
+//  Copyright (C) 2020 Schrödinger, LLC
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+#include <algorithm>
+#include <memory>
+
+#include <GraphMol/RDKitBase.h>
+
+#include "CIPLabeler.h"
+#include "CIPMol.h"
+#include "configs/Sp2Bond.h"
+#include "configs/Tetrahedral.h"
+
+#include "rules/Rules.hpp"
+#include "rules/Rule1a.h"
+#include "rules/Rule1b.h"
+#include "rules/Rule2.h"
+#include "rules/Rule3.h"
+#include "rules/Rule4a.h"
+#include "rules/Rule4b.h"
+#include "rules/Rule4c.h"
+#include "rules/Rule5New.h"
+#include "rules/Rule6.h"
+
+namespace RDKit {
+namespace CIPLabeler {
+
+namespace {
+
+// constitutional rules
+const Rules constitutional_rules({new Rule1a, new Rule1b, new Rule2});
+
+// all rules (require aux calc)
+const Rules all_rules({new Rule1a, new Rule1b, new Rule2, new Rule3, new Rule4a,
+                       new Rule4b, new Rule4c, new Rule5New, new Rule6});
+
+std::vector<std::unique_ptr<Configuration>> findConfigs(const CIPMol &mol) {
```

Comment:
I think we're going to need a version of this that takes a "list" of the atoms and bonds to be considered. 
We can then call it with the results of the new "findPossibleStereoAtomsAndBonds()" function. This function doesn't actually exist yet, but I'm working on it.

By "list" here I mean either a vector of ids to consider or, perhaps smarter, a dynamic_bitset with the appropriate bits set.

Reply:
Ok, I added an overload of the labeling function that takes two dynamic bitsets to specify which atoms and/or bonds should be examined.